### DISCUSSION TITLE
Adding torrent with 'Search Torrent' changed for the worse

### DISCUSSION THREAD

**Discussion by aarontemp123** [VueTorrent/VueTorrent#1006] [Q&A]
When I tried to add torrents via the 'Search Torrent' it used to be an overlay that popped up like when you click the + icon.
Now it brings up a new screen or full screen overlay and doesn't automatically put your cursor in the search bar.
Then Pressing enter after entering search pattern doesn't work either.

Then usually if you want to change your search pattern you have to go out of the search overlay and go back and do it all over again. very annoying if you need to download multiple episodes in the same season e.g.

Is this same for everyone or is mine just bugged out, have restarted my pi multiple times to no avail.
Vue version: 1.5.10, Qbittorrent: 4.5.4

**Comment by Larsluph** [comment]
Please update your WebUI. From what I understood it has already been fixed.

**Reply by aarontemp123** [reply]
I'm using Hotio's image. I thought it would automatically pull the latest versions of qbittorrent and vue whenever it builds the container?

**Reply by Larsluph** [reply]
The latest version is 1.6.1 and you said using 1.5.10 so it's not up to date...
